Conference-Room-Reservation-System/Conference-Room-Reservation-System/src/main/java/com/sda/conferenceroomreservationsystem/rest/ConferenceRoomController.java

```diff
@@ -22,30 +23,29 @@ public class ConferenceRoomController {
 
     @GetMapping("/all")
     public ResponseEntity<List<ConferenceRoomDto>> getAllConferenceRooms() {
-        List<ConferenceRoomDto> conferenceRooms = conferenceRoomService.getAllConferenceRooms;
-        return new ResponseEntity<>(conferenceRooms, HttpStatus.OK);
+        return ResponseEntity.ok(conferenceRoomService.getAll());
     }
 
-    @GetMapping("/find/{name}")
-    public ResponseEntity<ConferenceRoomDto> getConferenceRoomById(@PathVariable("name") final String name)
+    @GetMapping("/find/{id}")
+    public ResponseEntity<ConferenceRoomDto> getConferenceRoomById(@PathVariable("id") final Long id)
             throws ConferenceRoomNotFoundException
     {
-        return ResponseEntity.ok(conferenceRoomService.getConferenceRoomByName(name));
+        return ResponseEntity.ok(conferenceRoomService.getConferenceRoom(id));
     }
 
     @PostMapping("/add")
-    public ResponseEntity<ConferenceRoomDto> addConferenceRoom(@RequestBody final ConferenceRoomDto conferenceRoomDto)
```

Comment:
I think it would be better to keep the return of ConferenceRoomDto instead of ConferenceRoom - using mapper in service class - as DTO pattern stands.